sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UninvokedEventDeclaration.cs

```diff
@@ -60,41 +65,41 @@ private void RaiseOnUninvokedEventDeclaration(SymbolAnalysisContext context)
 
             var removableDeclarationCollector = new RemovableDeclarationCollector(namedType, context.Compilation);
 
-            var removableEvents = removableDeclarationCollector.GetRemovableDeclarations(
-                ImmutableHashSet.Create(SyntaxKind.EventDeclaration), maxAccessibility);
-            var removableEventFields = removableDeclarationCollector.GetRemovableFieldLikeDeclarations(
-                ImmutableHashSet.Create(SyntaxKind.EventFieldDeclaration), maxAccessibility);

```

Comment:
We found that ImmutableHashSet is a bit slower than HashSet before, hence I switched to HashSet.